vignettes/release_strategy.Rmd

```diff
@@ -43,17 +43,17 @@ updates via the community meetings, especially any anticipated breaking changes.
 
 A package release is done in five parts:
 
-1) Create a Pull Request from `devel` into the `pre-release` branch.  Issues identified in this Pull Request should have work done in separate branches and merged into the `pre-release` branch and **NOT** `devel`.
-1) Verify that all CI/CD checks are passing for the `devel` into the `pre-release` Pull Request, merge and then bundle up and send off to CRAN.
-1) Once the package is available on CRAN, another Pull Request is created for merging the `pre-release` branch into the `main` branch.  This will trigger the GitHub action to rebuild the `{admiral}` website with all the updates for this release.
-1) Use the release button on GitHub to "release" the package onto GitHub.  This release onto Github archives the version of code within the `main` branch, attaches the News/Changelog file, bundles the code into a `tar.gz` file and makes a validation report via the GitHub action `validation` from [insightsengineering/validatoR](https://github.com/insightsengineering/thevalidatoR).  Please see past [admiral releases](https://github.com/pharmaverse/admiral/releases) for reference. 
-1) Any issues fixed in the `pre-release/main` branches should be merged back into `devel`. 
+1) Create a Pull Request from `devel` into the `main` branch.  Issues identified in this Pull Request should have work done in separate branches and merged once again into `devel`.
+1) Verify that all CI/CD checks are passing for the `devel` into the `main` Pull Request, merge and then bundle up and send off to CRAN.
+1) If CRAN asks for modifications, repeat steps 1-2 as necessary.
+1) Once the package is accepted and available on CRAN, a GitHub action is set up to rebuild the `{admiral}` website with all the updates for this release.
+1) Use the release button on GitHub to "release" the package onto GitHub.  This release onto Github archives the version of code within the `main` branch, attaches the News/Changelog file, bundles the code into a `tar.gz` file and makes a validation report via the GitHub action `validation` from [insightsengineering/validatoR](https://github.com/insightsengineering/thevalidatoR).  Please see past [admiral releases](https://github.com/pharmaverse/admiral/releases) for reference and the [Releasing to Github](release_strategy.html#releasing-to-github) section for more details.
+1) Any issues fixed in the `main` branches should be merged back into `devel`. 
 1) Once a release is completed the `main` branch should be merged into `patch` to be ready for hotfixes.
 
 **Quarterly Release:**  
 
-* `devel >> pre-release >> main` 
-* `pre-release >> main >> devel` (Fixes done after a CRAN rejection)
+* `devel >> main` 
 * `main >> patch` (To be prepared in case of a needed hotfix)
```

Comment:
I think an automated check PR will be developed whenever devel goes into main.  Can this be noted as a future upcoming feature to be on the look out for